client/src/components/Map/index.tsx

```diff
@@ -115,6 +115,19 @@ function Map({ forLanding = false }: MapProps): React.ReactElement {
           <MapLi key={i} superBlock={superBlock} landing={forLanding} />
         ))}
       </ul>
+      {showUpcomingChanges && (
+        <>
+          <Spacer size='medium' />
+          <h1 className={forLanding ? 'big-heading' : ''}>
+            {t('landing.upcoming-heading')}
+          </h1>
```

Comment:
this should be `h2`, the page has multiple `h1`.

```suggestion
          <h2 className={forLanding ? 'big-heading' : ''}>
            {t('landing.upcoming-heading')}
          </h2>
```

Reply:
All the other headings are using h1 on this page. I think we should keep this as an h1 for now and make an issue about changing them all if that's what we want.

Reply:
There is a [PR](https://github.com/freeCodeCamp/freeCodeCamp/pull/51406) that change the structure for the page.